spec/features/building_map_spec.rb

```diff
@@ -46,32 +46,41 @@
     end
 
     it "shows the pin of a target point", js: true do
-      visit building_map_path(target: "52.393913,13.133082")
+      visit building_map_path(target: "52.393913,13.133082") # TODO: This doesn't work anymore
```

Comment:
Is there an issue that tracks this after this is merged?
in the meantime, you should probably skip this test.

Reply:
I added Issue #148 to track this problem